modules/nf-core/maaslin2/tests/main.nf.test

```diff
@@ -0,0 +1,74 @@
+nextflow_process {
+
+    name "Test Process MAASLIN2"
+    script "../main.nf"
+    process "MAASLIN2"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "maaslin2"
+
+    test("Normal run") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test_sample' ], // meta map
+                    file(params.modules_testdata_base_path + 'differentialabundance/modules_testdata/SRP254919.salmon.merged.gene_counts.top1000cov.assay.tsv', checkIfExists: true)
```

Comment:
You can directly use the following.
I don't know if htpps path would accept `../../`
```suggestion
                    file('https://raw.githubusercontent.com/nf-core/test-datasets/differentialabundance/modules_testdata/SRP254919.salmon.merged.gene_counts.top1000cov.assay.tsv', checkIfExists: true)
```